// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/conformance/declarationEmit/typeofImportTypeOnlyExport.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -5,8 +5,8 @@
 //// https://sokra.github.io/source-map-visualization#base64,aW1wb3J0IHsgQ2xhc3NNYXBEaXJlY3RpdmUgfSBmcm9tICcuL2xpdC5qcyc7DQpleHBvcnQgZGVjbGFyZSBjb25zdCBjOiB7DQogICAgZGlyZWN0aXZlOiBDbGFzc01hcERpcmVjdGl2ZTsNCn07DQovLyMgc291cmNlTWFwcGluZ1VSTD1idXR0b24uZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiYnV0dG9uLmQudHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJidXR0b24udHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsT0FBTyxFQUFDLGlCQUFpQixFQUFXLE1BQU0sVUFBVSxDQUFDO0FBQ3JELGVBQU8sTUFBTSxDQUFDLEVBQUU7SUFDWixTQUFTLEVBQUUsaUJBQWlCLENBQUM7Q0FDbkIsQ0FBQyJ9,aW1wb3J0IHtDbGFzc01hcERpcmVjdGl2ZSwgY2xhc3NNYXB9IGZyb20gJy4vbGl0LmpzJzsKZXhwb3J0IGNvbnN0IGM6IHsKICAgIGRpcmVjdGl2ZTogQ2xhc3NNYXBEaXJlY3RpdmU7Cn0gPSBjbGFzc01hcCgpOwo=
 
 
 //// [/.src/lit.d.ts.map]
-{"version":3,"file":"lit.d.ts","sourceRoot":"","sources":["lit.ts"],"names":[],"mappings":"AAAA,cAAM,iBAAiB;CAAG;AAE1B,YAAY,EAAC,iBAAiB,EAAC,CAAC;AAEhC,eAAO,MAAM,SAAS,cACR,CAAC,KAAG,MAAM;IAClB,WAAW,CAAC,CAAC;CAIf,CAAC;AAEL,eAAO,MAAM,QAAQ,EAAE,MAAM;IACzB,SAAS,EAAE,OAAO,iBAAiB,CAAC;CACR,CAAC"}
+{"version":3,"file":"lit.d.ts","sourceRoot":"","sources":["lit.ts"],"names":[],"mappings":"AAAA,cAAM,iBAAiB;CAAG;AAE1B,YAAY,EAAC,iBAAiB,EAAC,CAAC;AAEhC,eAAO,MAAM,SAAS,GACnB,CAAC,EAAE,MAAM,EAAE,CAAC,KAAG,MAAM;IAClB,SAAS,EAAE,CAAC,CAAC;CAIf,CAAC;AAEL,eAAO,MAAM,QAAQ,EAAE,MAAM;IACzB,SAAS,EAAE,OAAO,iBAAiB,CAAC;CACR,CAAC"}
 
-//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjbGFzcyBDbGFzc01hcERpcmVjdGl2ZSB7DQp9DQpleHBvcnQgdHlwZSB7IENsYXNzTWFwRGlyZWN0aXZlIH07DQpleHBvcnQgZGVjbGFyZSBjb25zdCBkaXJlY3RpdmU6IDxDPihjbGFzc186IEMpID0+ICgpID0+IHsNCiAgICBkaXJlY3RpdmU6IEM7DQp9Ow0KZXhwb3J0IGRlY2xhcmUgY29uc3QgY2xhc3NNYXA6ICgpID0+IHsNCiAgICBkaXJlY3RpdmU6IHR5cGVvZiBDbGFzc01hcERpcmVjdGl2ZTsNCn07DQovLyMgc291cmNlTWFwcGluZ1VSTD1saXQuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoibGl0LmQudHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJsaXQudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsY0FBTSxpQkFBaUI7Q0FBRztBQUUxQixZQUFZLEVBQUMsaUJBQWlCLEVBQUMsQ0FBQztBQUVoQyxlQUFPLE1BQU0sU0FBUyxjQUNSLENBQUMsS0FBRyxNQUFNO0lBQ2xCLFdBQVcsQ0FBQyxDQUFDO0NBSWYsQ0FBQztBQUVMLGVBQU8sTUFBTSxRQUFRLEVBQUUsTUFBTTtJQUN6QixTQUFTLEVBQUUsT0FBTyxpQkFBaUIsQ0FBQztDQUNSLENBQUMifQ==,Y2xhc3MgQ2xhc3NNYXBEaXJlY3RpdmUge30KCmV4cG9ydCB0eXBlIHtDbGFzc01hcERpcmVjdGl2ZX07CgpleHBvcnQgY29uc3QgZGlyZWN0aXZlID0KICA8Qz4oY2xhc3NfOiBDKTogKCkgPT4gewogICAgICBkaXJlY3RpdmU6IEM7CiAgfSA9PgogICgpID0+ICh7CiAgICBkaXJlY3RpdmU6IGNsYXNzXywKICB9KTsKCmV4cG9ydCBjb25zdCBjbGFzc01hcDogKCkgPT4gewogICAgZGlyZWN0aXZlOiB0eXBlb2YgQ2xhc3NNYXBEaXJlY3RpdmU7Cn0gPSBkaXJlY3RpdmUoQ2xhc3NNYXBEaXJlY3RpdmUpOwo=
+//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjbGFzcyBDbGFzc01hcERpcmVjdGl2ZSB7DQp9DQpleHBvcnQgdHlwZSB7IENsYXNzTWFwRGlyZWN0aXZlIH07DQpleHBvcnQgZGVjbGFyZSBjb25zdCBkaXJlY3RpdmU6IDxDPihjbGFzc186IEMpID0+ICgpID0+IHsNCiAgICBkaXJlY3RpdmU6IEM7DQp9Ow0KZXhwb3J0IGRlY2xhcmUgY29uc3QgY2xhc3NNYXA6ICgpID0+IHsNCiAgICBkaXJlY3RpdmU6IHR5cGVvZiBDbGFzc01hcERpcmVjdGl2ZTsNCn07DQovLyMgc291cmNlTWFwcGluZ1VSTD1saXQuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoibGl0LmQudHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJsaXQudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsY0FBTSxpQkFBaUI7Q0FBRztBQUUxQixZQUFZLEVBQUMsaUJBQWlCLEVBQUMsQ0FBQztBQUVoQyxlQUFPLE1BQU0sU0FBUyxHQUNuQixDQUFDLEVBQUUsTUFBTSxFQUFFLENBQUMsS0FBRyxNQUFNO0lBQ2xCLFNBQVMsRUFBRSxDQUFDLENBQUM7Q0FJZixDQUFDO0FBRUwsZUFBTyxNQUFNLFFBQVEsRUFBRSxNQUFNO0lBQ3pCLFNBQVMsRUFBRSxPQUFPLGlCQUFpQixDQUFDO0NBQ1IsQ0FBQyJ9,Y2xhc3MgQ2xhc3NNYXBEaXJlY3RpdmUge30KCmV4cG9ydCB0eXBlIHtDbGFzc01hcERpcmVjdGl2ZX07CgpleHBvcnQgY29uc3QgZGlyZWN0aXZlID0KICA8Qz4oY2xhc3NfOiBDKTogKCkgPT4gewogICAgICBkaXJlY3RpdmU6IEM7CiAgfSA9PgogICgpID0+ICh7CiAgICBkaXJlY3RpdmU6IGNsYXNzXywKICB9KTsKCmV4cG9ydCBjb25zdCBjbGFzc01hcDogKCkgPT4gewogICAgZGlyZWN0aXZlOiB0eXBlb2YgQ2xhc3NNYXBEaXJlY3RpdmU7Cn0gPSBkaXJlY3RpdmUoQ2xhc3NNYXBEaXJlY3RpdmUpOwo=
 
